CHANGELOG.md

```diff
@@ -3,14 +3,45 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0.0 - 2023-05-22
 
-Initial release of nf-core/rnasplice, created with the [nf-core](https://nf-co.re/) template.
+First release of nf-core/rnasplice, created with the [nf-core](https://nf-co.re/) template.
 
 ### `Added`
 
-### `Fixed`
+- Visualization of differential results with edgeR, DEXSeq, and MISO
+- Contrasts specified using contrastsheet.csv
+- Allow users to specify input data type and start point (e.g., fastq, genome_bam, transcript_bam, salmon_results)
+- Pipeline schematic updated
 
-### `Dependencies`
+## v1.0dev - [2022-11-10]
```

Comment:
I think that can be removed an all that section should go in the 1.0.0 release

Reply:
Fixed in #68